terraform/02_kube_worker_quick_spawn/00_instance/kube_worker_quick_spawn.tf

```diff
@@ -0,0 +1,20 @@
+resource "openstack_compute_instance_v2" "kube_worker_quick_spawn_tmp" {
+  name = "kube_worker-quick-spawn-tmp"

```

Comment:
why `-tmp` here?

Reply:
Because it's a temporary isntance. We want to spawn it, create a snapshot then remove it